raiden/transfer/state.py

```diff
@@ -23,6 +25,231 @@ def balanceproof_from_envelope(envelope_message):
     )
 
 
+class NodeState(State):
+    """ Umbrela object that stores all the node state.
+    For each registry smart contract there must be a payment network, withn the
+    payment network the existing token networks and channels are registered.
+    """
+
+    __slots__ = (
+        'block_number',
+        'identifiers_to_paymentnetworks',
+        'nodeaddresses_to_networkstates',
+        'payment_mapping',
+    )
+
+    def __init__(self, block_number: typing.block_number):
+        if not isinstance(block_number, typing.block_number):
+            raise ValueError('block_number must be an block_number')
+
+        self.block_number = block_number
+        self.identifiers_to_paymentnetworks = dict()
+        self.nodeaddresses_to_networkstates = dict()
+        self.payment_mapping = PaymentMappingState()
+
+    def __repr__(self):
+        return '<NodeState block:{} networks:{} qtd_transfers:{}>'.format(
```

Comment:
What does `qtd` stand for?

Reply:
Good questin. I also wonder. Maybe he wanted to say quantity? so `qty`? @hackaugusto ?

Reply:
Quantity makes sense

Reply:
Yep, it's quantity. `qtd` is the portuguese abbreviation

Reply:
ha nice! So `quantidade` -> `qtd`. Should rename it to `qty` for clarity eventually ;)